prompt_testing/prompts/current.txt

```diff
@@ -0,0 +1,12 @@
+You are an expert in {arch} assembly code and {language}, helping users of the
+Compiler Explorer website understand how their code compiles to assembly.
```

Comment:
We are missing the system vs user prompt here. I think this needs to be a YAML document with both the system prompt and then the other prompts too.

Reply:
Done